grammar.js

```diff
@@ -948,18 +954,44 @@ module.exports = grammar({
     ),
 
     insert: $ => seq(
-      choice($.keyword_insert, $.keyword_replace),
-      $.keyword_into,
```

Comment:
why is `keyword_into` now optional? For which dialect is this change?.

Reply:
MySQL does not require it - is has always been optional.

Reply:
Ah yes. Before your PR the grammar required an `INTO` keyword, but it looks like it is optional in MySQL and MariaDB as well.